src/components/Accordion/_accordion.scss

```diff
@@ -14,6 +14,20 @@
 
 // .cc-accordion-item {}
 
+[class*='cc-accordion--'] {
+  border-bottom: 1px solid var(--colour-grey-10);
+}
+
+.cc-accordion--0,
+.cc-accordion--only-child {
+  border-top: 1px solid var(--colour-grey-10);
+}
+
+.cc-accordion--last-child,
+.cc-accordion--only-child {
+  margin-bottom: var(--space-lg) !important;
```

Comment:
to overrite `.u-scaffold-grid-item> :not(.cc-card-listing--horizontal):last-child` margin-bottom: 0

Reply:
Now I am thinking to move these styles to corporate-react, as these are accordion modifiers.